### DISCUSSION TITLE
Keep getting timeout when saving to Google drive

### DISCUSSION THREAD

**Discussion by adelucca** [jgraph/drawio#3859] [General]
Running version 21.7.5 on Chrome. 

This is a file I have been working on for a few years already, so not sure if some connection draw.io <> google drive broke at some point. 

The file appears in my GDrive, I can create new sheets, and work on them, but when it tries to save it will fail 95% of the time. 
Somehow it works from time to time, but not most.
![image](https://github.com/jgraph/drawio/assets/26552954/73f706e3-df3e-451d-bffa-63e81d0147bc)

I![image](https://github.com/jgraph/drawio/assets/26552954/083748af-f05d-4dd0-a65d-51569e6d7c6d)

Most of the time it will fail to save (auto or manual), with this note persisting.
![image](https://github.com/jgraph/drawio/assets/26552954/96220aab-662d-4a93-b1c8-2a8497b33367)

I have looked on the internet and cannot find any applicable solution.
Please let me know what information I can provide to help troubleshoot.
![image](https://github.com/jgraph/drawio/assets/26552954/ee3dacbb-0318-44dd-8c8e-f663961a5600)
The file in question has ~20 sheets, and weights ~14mb - maybe the size is causing the issue?

Help please, it's very frustrating.

**Comment by davidjgraph** [comment]
Could well be the size. Are there a lot of embedded images?

**Reply by adelucca** [reply]
Thank you for the super quick attention!

Out of the 20 sheets, there are 5-6 sheets that have anywhere between 2 to 10 images. Most images are relatively small, but some images are full page captures of sites.
Rest of sheets no images.

I just tried to create a copy and failed.

Maybe I can try to create a new file, and see if the issue persists? If fixed, maybe start moving items to the new one.

**Reply by adelucca** [reply]
@davidjgraph FYI - I created a new file, started working on it, and I have no issues with saving. Bummer that now I have two files, but at least it seems my GDrive integration is not the issue. 

Thank you for looking into this
